_pages/about.fr

```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
 
 [/et_pb_slider][/et_pb_column][/et_pb_row][et_pb_row admin_label="Row" make_fullwidth="off" use_custom_width="off" width_unit="on" use_custom_gutter="on" gutter_width="4" padding_mobile="off" allow_player_pause="off" parallax="off" parallax_method="off" make_equal="off" column_padding_mobile="on"][et_pb_column type="1_2"][et_pb_text admin_label="Text" background_layout="light" text_orientation="left" use_border_color="off" border_color="#ffffff" border_style="solid"]
 
-Subsurface can <strong>track</strong> single- and multi-tank <strong>dives</strong> using air, Nitrox or TriMix. It allows tracking of dive locations including GPS coordinates (which can also conveniently be entered using a map interface), logging of equipment used and names of other divers, and lets users rate dives and provide additional notes. 
+Subsurface permet le <strong>suivi</strong> de <strong>plongées</strong> a l'air, au Nitrox ou au TriMix pour une ou plusieurs bouteilles. Il donne la possibilité du suivit des sites de plongées ainsi que leur position GPS (cette dernière peut être facilement saisie a l'aide d'une carte), de l'enregistrement de l'équipement utilisé, du nom des autres plongeurs, et de remarques supplémentaires ainsi que d'une notation des plongées.
```

Comment:
Il donne la possibilité d'enregistrer les sites de plongées..., l'équipement utilisé, le nom des autres plongeurs, noté les plongées et information supplémentaire désirée?